sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -854,6 +854,11 @@ ssize_t coap_opt_add_string(coap_pkt_t *pkt, uint16_t optnum, const char *string
     return write_len;
 }
 
+ssize_t coap_opt_add_path(coap_pkt_t *pkt, const char *path)
```

Comment:
Why not inline this implementation in the header?

Reply:
I let myself influence by the adjacent `coap_opt_add_uquery()` - fixed.